packages/typespec-test/test/schemaRegistry/generated/typespec-ts/src/rest/parameters.ts

```diff
@@ -10,16 +10,8 @@ export type ListSchemaVersionsParameters = RequestParameters;
 export type GetSchemaByVersionParameters = RequestParameters;
 
 export interface GetSchemaIdByContentBodyParam {
-  /**
-   * String representation (UTF-8) of the registered schema.
-   *
-   * Value may contain any sequence of octets
-   */
-  body:
-    | string
-    | Uint8Array
-    | ReadableStream<Uint8Array>
-    | NodeJS.ReadableStream;
+  /** String representation (UTF-8) of the registered schema. */
+  body: string;
```

Comment:
This is the fix for bytes. The defined media types are non-binary types so we should generate string not binary for it.

Reply:
Can you help me remember why we generate `string` for bytes and not `Uint8Array`?